public/iiif/speculative-annotations-manifest.json

```diff
@@ -53,6 +53,7 @@
       ],
       "partOf": [
         {
+          "id": "https://www.loc.gov/search/?fa=contributor:muybridge,+eadweard&sp=1",
```

Comment:
@jaimemears Here's the `id` field I was referring to in the other comment

Reply:
I see. I really dont know. I think whatever you think we should do is best. Second one seems easier?

Reply:
Gotcha.  Yea, I'll go route 2.